vars/toolValidate.groovy

```diff
@@ -1,31 +1,45 @@
 import com.sap.piper.FileUtils
 import com.sap.piper.Version
+import com.sap.piper.Utils
+import com.sap.piper.ConfigurationLoader
+
 import hudson.AbortException
 
 
 def call(Map parameters = [:]) {
```

Comment:
It is a very good pattern to list all the parameters used by the step here in a List (actually it would be better with a Set, but it is a list at the moment. Map would also be reasonable in case of a deep merge, but everything in parentheses is a side-battle). That set is used for filtering in conjunction with the parameter merging. I would like to recommand to follow this pattern also here.

Reply:
you can create a PR